modules/utxo-lib/test/integration_local_rpc/fixtures/bitcoinPublicSignet/v1/deposit_p2pkh.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
I assume these are all derived via the btc regtest node anyway

please change the test code so that we use the same fixtures for signet and testnet here

Reply:
I tried, but testnet addresses are e.g., `bcrt1q3vyfqm32xqwnmv0up47n9r00pch2qmz8se0ngw` using the fixtures, while public signet addresses are `t1q...`

Reply:
If the prefixes matter, I can do some digging to see why they're different

Reply:
Oh I see, the `testnet` fixtures are actually `regtest`